src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusTriggerExecutor.cs

```diff
@@ -24,9 +24,15 @@ public async Task<FunctionResult> ExecuteAsync(Message value, CancellationToken
             TriggeredFunctionData input = new TriggeredFunctionData
             {
                 ParentId = parentId,
-                TriggerValue = value
+                TriggerValue = value,
+                TriggerDetails = TriggerDetails(value)
             };
             return await _innerExecutor.TryExecuteAsync(input, cancellationToken);
         }
+
+        private string TriggerDetails(Message value)
+        {
+            return $"MessageId: {value.MessageId}, TimeToLive: {value.TimeToLive}, ScheduleEnqueueTime: {value.ScheduledEnqueueTimeUtc}";
```

Comment:
What about the details I suggested "MessageId: X, DeliveryCount: Y, EnqueuedTime: Z"? DeliveryCount would be useful. Also, we should use the EnqueuedTime from SystemProperties, not ScheduledEnqueueTime. Not sure if TTL would be useful - recommend replacing that with DeliveryCount (also from SystemProperties)

Reply:
I wasn't sure where to pull those out from so I used what looked like the closest equivalents. I'll add them in from the SystemProperties.